release-notes/opensearch-anomaly-detection-dashboards.release-notes-1.1.0.0.md

```diff
@@ -0,0 +1,31 @@
+## Version 1.1.0.0 Release Notes
+
+Compatible with OpenSearch Dashboards 1.1.0
+
+### Features
+
+* Add single flow functionality ([#63](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/pull/63))
```

Comment:
we miss compact rcf integration here.

Reply:
Not sure what you mean, this is for Dashboards

Reply:
oh, i mixed them up.